### ISSUE TITLE
[ENH] Build instructions

### ISSUE THREAD

**Issue by aalku** [jcefmaven/jcefmaven#93]
**Is your feature request related to a problem? Please describe.**
I am using jcefmaven as user and I am trying to contribute a tiny feature to jcef but there seem to be many steps from editing jcef to use the new release with jcefmaven so I can, at least, test it and I'm lost in them.

**Describe the solution you'd like**
I'd like instructions to make a custom release of jcefmaven so I can try my internal releases before publishing (pull-request).

**Describe alternatives you've considered**
I've considered to learn how to use jcef without jcefmaven but it doesn't look easy.

**Additional context**
I wish to use respose.setCharset() but but it isn't exposed. I'd like to try this changes I made: https://github.com/chromiumembedded/java-cef/compare/master...aalku:java-cef:master

**Comment by aalku** [comment]
I've made a Windows amd64 binary release with jcefbuild but I don't know how to use the binary release with jcefmaven. It seems I need to make a jcef-api release based on it and then a jcefmaven or maybe your scripts do both at once. I don't know how.

**Comment by FriwiDev** [comment]
Creating the artifact is relatively easy if you already have a jcefbuild. You simply start the jcefmaven pipeline, but you will need to point the pipeline to the download of your build info json file provided by jcefbuild.  You can remove all sonatype (maven) pushing steps of your jcefmaven fork to not run into build errors this way. Then you can use the normal jcefmaven jar and set a new download path for the natives to link to your just created natives on github (they appear in the release and can be downloaded from there).

**Comment by FriwiDev** [comment]
Please have a look at the source here to see how to provide the custom url: https://github.com/jcefmaven/jcefmaven/blob/master/jcefmaven/src/main/java/me/friwi/jcefmaven/CefAppBuilder.java#L76

**Comment by aalku** [comment]
> You simply start the jcefmaven pipeline

I'm sorry but I don't know what you mean. 😅 Or how to do it. If you mean running a script I don't know which one or with what arguments.

Thanks for your help <3

**Comment by FriwiDev** [comment]
Please see https://github.com/jcefmaven/jcefmaven/issues/93

**Comment by FriwiDev** [comment]
The github actions also only run a script. Basically they only run `https://github.com/jcefmaven/jcefmaven/blob/master/generate_artifacts.sh`, which you can run in WSL if you like :)

**Comment by FriwiDev** [comment]
I just suggest running with github actions, because it takes the hassle from you to have to set up components on your PC.

**Comment by aalku** [comment]
Hi! Thanks a lot! Still trying. 

When I said y had built jcefbuild I meant in my computer only for my architecture. Now I did it in GitHub so I got a release published so in theory I can run jcefmaven.

But there are some problems. I think changing the repository as you said [here](https://github.com/jcefmaven/jcefmaven/issues/93#issuecomment-1892399742) is not enough as there is a change in the API so I need my release of jcefmaven depends on my release of the API so my code compiles. I think the API jar is generated by the same pipeline.

So I started to make changes like this one:
![image](https://github.com/jcefmaven/jcefmaven/assets/12587056/2e367528-62fb-443e-8868-5236f4349411)

But I think I broke things. jcefmaven's pipeline is not working.

I'll undo the changes, get it to run and then maybe make some of them again.

Thank you for your help!

**Comment by aalku** [comment]
🥺 Without those changes it didn't build either.

I don't know what I'm doing wrong.

`generate-artifacts    | [ERROR] The specified file '/jcefout/jogl-all-v2.4.0-rc-20210111.jar' not exists`

The pipeline say it succeded but one error caused other errors so it didn't create the jcefmaven artifact.

**Comment by aalku** [comment]
Actually no. It didn't build right either. 😅

**Comment by aalku** [comment]
Ok, I got to build the API jar by hand and with that I made it work.
the `jcef-api-jcef-1dbf828+cef-119.4.7+g55e15c8+chromium-119.0.6045.199-sources.jar` (sources) file was ok but the generated `jcef-api-jcef-1dbf828+cef-119.4.7+g55e15c8+chromium-119.0.6045.199.jar` was 1KB only, with no classes. 🤷🏻‍♂️🥺

With that I could test my code. Finally!! And it works. So I'm going to send a pull request to jcef. 

Thanks again for your help!
